src/services/milestones/sendNotification.js

```diff
@@ -241,15 +241,15 @@ const sendNotification = () => async context => {
           message,
         });
       } else if (status === PAID && mined && prevStatus === PAYING) {
-        Notifications.milestonePaid(app, {
-          recipient: recipient.email,
-          user: recipient.name,
-          milestoneTitle: title,
-          milestoneId: _id,
-          donationCounters,
-          address: recipientAddress,
-          campaignId,
-        });
+        // Notifications.milestonePaid(app, {
```

Comment:
@aminlatifi 
I think these are should be delete but I commented it to know your opinion for that.

Reply:
Yes remove it

Reply:
Also remove milestonePaid function definition.

Reply:
> Also remove milestonePaid function definition.

Done.